app/bundles/ReportBundle/Controller/Api/ReportApiController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * @package     Mautic
+ * @copyright   2014 Mautic Contributors. All rights reserved.
+ * @author      Mautic
+ * @link        http://mautic.org
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\ReportBundle\Controller\Api;
+
+use FOS\RestBundle\Util\Codes;
+use JMS\Serializer\SerializationContext;
+use Mautic\ApiBundle\Controller\CommonApiController;
+use Symfony\Component\HttpKernel\Event\FilterControllerEvent;
+
+/**
+ * Class ReportApiController
+ */
+class ReportApiController extends CommonApiController
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    public function initialize(FilterControllerEvent $event)
+    {
+        parent::initialize($event);
+        $this->model            = $this->factory->getModel('report');
+        $this->entityClass      = 'Mautic\ReportBundle\Entity\Report';
+        $this->entityNameOne    = 'report';
+        $this->entityNameMulti  = 'reports';
+        $this->permissionBase   = 'report:reports';
+        $this->serializerGroups = array('reportList', 'reportDetails');
+    }
+
+    /**
+     * Obtains a list of reports
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function getEntitiesAction()
+    {
+        if (!$this->security->isGranted('report:reports:viewother')) {
+            $this->listFilters = array(
+                'column' => 'r.createdBy',
+                'expr'   => 'eq',
+                'value'  => $this->factory->getUser()
```

Comment:
This probably should be ...getUser()->getId() as the entity createdBy properties are no longer associated with User entities but just the user ID (int).